deny.toml

```diff
@@ -1,14 +1,12 @@
 [advisories]
 db-path = "~/.cargo/advisory-db"
 db-urls = ["https://github.com/rustsec/advisory-db"]
-vulnerability = "deny"
-unmaintained = "warn"
+# https://github.com/EmbarkStudios/cargo-deny/pull/611
+version = 2
 yanked = "deny"
-notice = "warn"
 
 
 [licenses]
```

Comment:
Do we also have to add a `version = 2` field to the `[licenses]`? It seems Embark did the same in https://github.com/EmbarkStudios/cargo-deny/pull/611.